var/spack/repos/builtin/packages/scorep/package.py

```diff
@@ -72,6 +74,7 @@ def install(self, spec, prefix):
                 "--with-papi-header=%s" % spec['papi'].prefix.include,
                 "--with-papi-lib=%s" % spec['papi'].prefix.lib,
                 "--enable-shared",
+                "--without-shmem" if '~shmem' in spec else '',
```

Comment:
That would definitely be useful. Although I seem to recall @alalazo already having done that in one of his many PRs.